apps/cloudinary/src/index.js

```diff
@@ -72,17 +72,22 @@ function makeThumbnail(resource, config) {
 function renderDialog(sdk) {
   const { cloudinary } = window;
   const config = sdk.parameters.invocation;
-  let default_transformations = {};
+  let defaultTransformations = JSON.parse(config.defaultTransformations);
```

Comment:
Can we safely assume that this is not failing when we are testing it in `validateParameters` below?

Reply:
I don't think so. There is no validation on the free-form symbol type